### ISSUE TITLE
It dont scan good

### ISSUE THREAD

**Issue by Neroxm** [bitbrute/evillimiter#73]
It really dont scan good because the ip adresses on the scan are not goodl . How can i fix this? sorry for my english

**Comment by bitbrute** [comment]
Are the scanned IP addresses invalid or are there addresses missing?

**Comment by Neroxm** [comment]
> Are the scanned IP addresses invalid or are there addresses missing?

yes, they are invalid, i dont know if i need the wifi adapter so maybe its because i need the adapter.
